src/system/NewForm/FormAutocomplete.js

```diff
@@ -233,8 +248,9 @@ const FormAutocomplete = React.forwardRef(
 						label={ inlineLabel ? <SelectLabel /> : null }
 					>
 						{ searchIcon && <FormSelectSearch /> }
+
 						<Autocomplete
-							id={ id }
+							id={ forLabel }
```

Comment:
What made you prefer using `forLabel` for this? 
I'm worried we might risk some clashes in IDs because, when filling the `id` I already know it should be unique, whereas the `forLabel` doesn't convey this idea

Reply:
@andrea-sdl All other form components uses `forLabel`, I am actually putting the standard. Changing everything to ID might be a hard task on the VIP Dashboard side.